app/views/pages/about.html.erb

```diff
@@ -1,16 +1,10 @@
 <h1>About</h1>
 
-<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac
-turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor
-sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies
-mi vitae est. Mauris placerat eleifend leo.</p>
+<p>outcomes.mit.edu is an application designed for tracking student learning
+outcomes for ABET Accreditation for the School of Engineering.</p>
 
-<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac
-turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor
-sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies
-mi vitae est. Mauris placerat eleifend leo.</p>
+<p>Access is managed via the Roles Database, and can be assigned by your
+department's HR contact.</p>
 
-<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac
-turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor
-sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies
-mi vitae est. Mauris placerat eleifend leo.</p>
+<p>For more information please contact Jon Daries in Institutional Research at
+<a href="mailto:daries@mit.edu">daries@mit.edu</a></p>
```

Comment:
You can also use the Rails helper: `<%= mail_to "daries@mit.edu" %>`

http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-mail_to